native/NativeSampleApps/SmartSyncExplorer/src/com/salesforce/samples/smartsyncexplorer/ui/MainActivity.java

```diff
@@ -132,6 +132,15 @@ protected void onCreate(Bundle savedInstanceState) {
 	}
 
 	@Override
+	protected void logoutCompleteActions() {
+		super.logoutCompleteActions();
```

Comment:
Let me merge that PR with the recreate in the sample app code.
I'll do more testing before making the change in the SDK activity classes to see if it doesn't have unforeseen side effects in hybrid / react native apps.